src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -24,7 +25,7 @@ export class MdTabNavBar {
 @Directive({
   selector: '[md-tab-link]',
 })
-export class MdTabLink {
+export class MdTabLink extends MdRipple {
```

Comment:
I agree, it's odd to say that the md tab link extends a ripple. I'd prefer to keep this an attribute directive to preserve the end user's HTML using all native elements (nav, a), e.g.:

```
<nav md-tab-nav-bar>
  <a md-tab-link>Link</a>
  <a md-tab-link>Link</a>
</nav>
```

So there are two more alternatives:
1. Add [md-tab-link] selector to the MdRipple
2. Create a new MdTabLinkRipple directive that will only extend MdRipple and match [md-tab-link]

In talking with Jeremy, I think (2) sounds better than MdTabLink extending MdRipple and (1) just sounds more silly than either.